app/Models/DatabaseHost.php

```diff
@@ -57,6 +64,7 @@ class DatabaseHost extends Model
         'password' => 'nullable|string',
         'node_ids' => 'nullable|array',
         'node_ids.*' => 'required|integer,exists:nodes,id',
+        'driver' => 'required|string',
```

Comment:
```suggestion
        'driver' => 'required|string|in:' . self::DRIVERS,
```
`public const DRIVERS = 'mysql,mariadb,pgsql';`

Reply:
There's an enum rule specifically